src/components/Alert/Alerts.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import {Alert} from 'react-bootstrap';
+import styles from './Alert.module.scss';
+
+interface Props {
+  type: string;
+}
+
+export function Alerts ({type}: Props) {
+
+    const [show, setShow] = useState(true);
+
+  if (type.match(`warning`)) {
```

Comment:
string comparisons can be done just by using the `==` symbol.
```suggestion
  if (type == 'warning') {
```